src/app/loading.tsx

```diff
@@ -1,7 +1,11 @@
 import React from "react";
 
-const Loading: React.FC<{}> = () => {
-    return <h1> Loading </h1>;
+const Loading: React.FC = () => {
+    return (
+        <main>
```

Comment:
why is this being changed? It's a good change, but seemingly out of nowhere

Reply:
Was to fix accessibility tests, but was really just a workaround as I for some reason didn't realise this meant the accessibility tests was testing the loading page. Have now added code to the tests to ensure the login page fully loads.